includes/class-wc-calypso-bridge-woocommerce-admin.php

```diff
@@ -49,6 +53,38 @@ public function maybe_remove_devdocs_menu_item( $features ) {
 
 		return $features;
 	}
+
+	/**
+	 * Remove the legacy `WooCommerce > Coupons` menu.
+	 *
+	 * @param mixed $value Value to be filtered.
+	 *
+	 * @return int 1 to show the legacy menu, 0 to hide it. Booleans do not work.
+	 * @see  Automattic\WooCommerce\Internal\Admin\CouponsMovedTrait::display_legacy_menu()
+	 * @todo Write a compatibility branch in CouponsMovedTrait to hide the legacy menu in new installations of WooCommerce.
+	 */
+	public function filter_show_legacy_coupon_menu( $value ) {
+		return 0;
+	}
+
+	/**
+	 * Delete all existing `Coupon Page Moved` notes from the DB.
+	 *
+	 * @return void
+	 * @todo Create a one-time operation controller, to delete all `wc-admin-coupon-page-moved` notes from the database.
+	 */
+	public function delete_coupon_page_moved_notes() {
+
+		if ( class_exists( 'Automattic\WooCommerce\Admin\Notes\Notes' ) ) {
```

Comment:
You can always trust that the latest WC version will be available in the .com environment.

Reply:
Cool cool cool cool cool cool